src/blocks/block-coauthors/edit.js

```diff
@@ -0,0 +1,243 @@
+		} );
+	}
+
+	const layoutControls = [
+		{
+			icon: list,
+			title: __( 'Inline' ),
+			onClick: () => setLayout( { type: 'inline' } ),
+			isActive: layout.type === 'inline',
+		},
+		{
+			icon: grid,
+			title: __( 'Block' ),
+			onClick: () =>
+				setLayout( { type: 'block' } ),
+			isActive: layout.type === 'block',
+		},
+	];
+
+	return (
+		<>
+			<BlockControls>
+				<ToolbarGroup controls={ layoutControls } />
+				<AlignmentControl
+					value={ textAlign }
+					onChange={ ( nextAlign ) => {
+						setAttributes( { textAlign: nextAlign } );
+					} }
+				/>
+			</BlockControls>
+			<div { ...useBlockProps({
+					className: classnames( {
+						[`is-layout-cap-${layout.type}`]: layout.type,
+						[`has-text-align-${ textAlign }`]: textAlign,
+					}),
+					style: {
+						gap: __experimentalGetGapCSSValue( attributes?.style?.spacing?.blockGap )
+					}
+				})
+			}>
+				{
+					coAuthors &&
+					'inline' === layout.type &&
+					( isSelected || prefix ) &&
+					(
+						<RichText
+							allowedFormats={ ALLOWED_FORMATS }
+							className="wp-block-cap-coauthors__prefix"
+							multiline={ false }
+							aria-label={ __( 'Prefix' ) }
+							placeholder={ __( 'Prefix' ) + ' ' }
+							value={ prefix }
+							onChange={ ( value ) =>
+								setAttributes( { prefix: value } )
+							}
+							tagName="span"
+						/>
+					)
+				}
+				{
+					coAuthors && 
+					coAuthors
+					.map( ( author ) => {
+						const isHidden = author.id === ( activeBlockContextId || coAuthors[0]?.id );
+						return (
+							<BlockContextProvider
+								key={ author.id }
+								value={ {'cap/author': author } }
+							>
+								{ isHidden ? (<CoAuthorTemplateInnerBlocks />) : null }
+								<MemoizedCoAuthorTemplateBlockPreview
+									blocks={blocks}
+									blockContextId={author.id}
+									setActiveBlockContextId={ setActiveBlockContextId }
+									isHidden={isHidden}
+								/>
+							</BlockContextProvider>
+						);
+					})
+					.reduce( ( previous, current, index, all ) => (
+						<>
+						{ previous }
+						{
+							'inline' === layout.type &&
+							(
+								<span className="wp-block-cap-coauthors__separator">
+									{ ( lastSeparator && index === (all.length - 1) ) ? `${lastSeparator}` : `${separator}` }
+								</span>
+							)	
+						}
+						{ current }
+						</>
+					))
+				}
+				{
+					coAuthors &&
+					'inline' === layout.type &&
+					( isSelected || suffix ) &&
+					(
+						<RichText
+							allowedFormats={ ALLOWED_FORMATS }
+							className="wp-block-cap-coauthors__suffix"
+							multiline={ false }
+							aria-label={ __( 'Suffix' ) }
+							placeholder={ __( 'Suffix' ) + ' ' }
+							value={ suffix }
+							onChange={ ( value ) =>
+								setAttributes( { suffix: value } )
+							}
+							tagName="span"
+						/>
+					)
+				}
+			</div>
+			<InspectorControls>
+				{
+					'inline' === layout.type &&
+					(
+						<PanelBody title={ __( 'CoAuthors Layout' ) }>
```

Comment:
```suggestion
						<PanelBody title={ __( 'Co-authors Layout' ) }>
```

Reply:
Addressed in https://github.com/Automattic/Co-Authors-Plus/pull/997/commits/52dc33dcae7a53317e61d48b679b292928f4c56b